proto/claim/tx.proto

```diff
@@ -6,8 +6,16 @@ option go_package = "github.com/ignite/modules/x/claim/types";
 // Msg defines the Msg service.
 service Msg {
   rpc ClaimInitial(MsgClaimInitial) returns (MsgClaimInitialResponse);
+  rpc Claim(MsgClaim) returns (MsgClaimResponse);
 }
 
-message MsgClaimInitial { string claimer = 1; }
+message MsgClaimInitial {string claimer = 1;}
 
 message MsgClaimInitialResponse {}
+
+message MsgClaim {
+  string claimer = 1;
+  uint64 missionID = 2;
+}
+
+message MsgClaimResponse {}
```

Comment:
Maybe we could consider adding the amount of claimed tokens as it is the result of the state transition

Simplifying building a client that give the total number of token received in a UI for example